lib/bucky/test_equipment/verifications/status_checker.rb

```diff
@@ -34,7 +34,12 @@ def http_status_check(args)
           # If number of requests is over redirect limit
           return { error_message: "\n[Redirect Error] #{url} is redirected more than #{REDIRECT_LIMIT}" } if redirect_count > REDIRECT_LIMIT
 
-          check_result = check_log_and_get_response(url, device, link_check_max_times, url_log)
+          begin
+            check_result = check_log_and_get_response(url, device, link_check_max_times, url_log)
+          rescue Net::ReadTimeout => e
+            puts "  #{url} ... ReadTimeout"
```

Comment:
Is this stdout needed?
or How about including this message into return error message?

Reply:
It shows the message on console just like status 200 show OK, status 400 show NG.
Just follow the implemented exist code style. It's OK for not showing this message.
And it's already showing in the error message. (by line 41.)